opdracht-1/styles.css

```diff
@@ -9,33 +9,44 @@ body {
 }
 
 /* 1.1 Geef de 'img' tag in the header een ID en zet zijn breedte op 100% */
-
-
+#image-header{
+    width: 100%;
+}
 
 /* 1.2 Verander de kleur van alle list items in de unordered list naar hsl(110, 29%, 52%) */
-
-
-
+ul > li {
+    color: hsl(110, 29%, 52%);
+}
 /* 1.3 Geef de 'span' tag in de footer een beschrijvende class name en maak de lettergrootte kleiner  */
-
-
-
+#trade-mark{
+    font-size: 4px;
+}
 /* 1.4 Haal de text-decoration van navigatie-links weg, geef onbezochte links de kleur hsl(359, 79%, 56%) en verander de kleur wanneer de gebruiker er overheen hovert. */
-
-
-
+nav a {
+    text-decoration: none;
+}
+nav a:link{
+    color: hsl(359, 79%, 56%);
+}
+nav a:hover{
+    color: purple;
+}
 /* 1.5 Maak een hover- en visited stijl voor alle links in 'main'. */
-
-
-
+main a:hover{
+    color: purple;
+}
+main a:visited{
+    color:deeppink;
+}
 /* Verander de achtergrond van de SignUp button. Tip: https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors  */
-
-
-
+div [value="Signup"]{
+    background-color: magenta;
+}
 /* Geef het tekst-veld een blauwe rand wanneer hij in focus is */
-
-
-
+.input[type=text]:focus {
```

Comment:
46: deze regel werkt niet bij mij. De rand moet blauw worden maar hij wordt groen. 
Een aantal dingen:
Je spreekt de input hier aan met class “.” maar dat kan je weglaten want het is gewoon een element
[input=text] moet zijn [input=”text”]
Je kan :focus weglaten volgens mij
De properties op 47 & 48 werken niet, volgens mij is de juiste property outline-color: blue;